analytic_engine/src/compaction/scheduler.rs

```diff
@@ -78,6 +79,7 @@ impl Default for SchedulerConfig {
             // flush_interval default is 5h.
             max_unflushed_duration: ReadableDuration(Duration::from_secs(60 * 60 * 5)),
             memory_limit: ReadableSize::gb(4),
+            max_pending_compaction_tasks: MAX_PENDING_COMPACTION_TASKS,
```

Comment:
I think `const` is unnecessary here.

Reply:
well, I will remove MAX_PENDING_COMPACTION_TASKS and MAX_ONGOING_TASKS.